tests/integration/butterfree/load/test_sink.py

```diff
@@ -47,8 +46,8 @@ def test_sink(input_dataframe, feature_set):
     sink.flush(feature_set, feature_set_df, client)
 
     # get historical results
-    historical_result_df = client.read_table(
-        feature_set.name, historical_writer.database
+    historical_result_df = client.read(
+        s3config.format_, options=s3config.get_options(feature_set.name)
     )
```

Comment:
After the merge of the PR adding the partitions to the table, I think this PR needs to come back to read from table, not directly from files. What the Butterfree expects is that after writing the data will be available in to be queried on the table at the Metastore. What do you think? Maybe adding a #TODO here or remember to add this detail to the `Writers` refactoring PR.